.github/workflows/manual-deploy.yml

```
@@ -0,0 +1,317 @@
+    environment: deployer
+
+    steps:
+      - name: Validate Version Format
+        run: |
+          if [[ ! "${{ inputs.version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Error: Version must be in format x.y.z (e.g., 1.2.3)"
+            exit 1
+          fi
+
+      - name: Checkout Project
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)